src/main/java/mops/foren/domain/model/User.java

```diff
@@ -0,0 +1,23 @@
+package mops.foren.domain.model;
+
+import javax.validation.constraints.Email;
+import java.util.HashMap;
+import java.util.List;
+
+public class User {
+    private String name;
+    private Email email;
+    private HashMap<Long, Permission> permissons;
```

Comment:
In the branch "permissions-and-new-domain-model" I got to implement a permission-system which suits this storage of permissions inside of a HashMap. The next step would be to implement a PermissionService, which checks the user-permission on security-related events.